app/controllers/hiring_staff/vacancies/copy_controller.rb

```diff
@@ -0,0 +1,25 @@
+class HiringStaff::Vacancies::CopyController < HiringStaff::Vacancies::ApplicationController
+  def create
+    vacancy = Vacancy.find(vacancy_id)
+    vacancy_copy = vacancy.dup
+    update_fields(vacancy_copy)
+    if vacancy_copy.save
+      redirect_to review_path(vacancy_copy)
```

Comment:
Before we hand over to the review and existing publish code, I think it would be sensible to start collecting audit data here as soon as its shipped.

`Auditor::Audit.new(vacancy_copy, 'vacancy.copy', current_session_id).log`

This will at least make this feature deployable while we work out how we're going to collect this going forward, and allow us to backfill data reliably.